f2/src/summary/HowDidItStartSummary.js

```diff
@@ -68,13 +67,19 @@ export const HowDidItStartSummary = (props) => {
           <H2 fontWeight="normal">
             <Trans id="confirmationPage.howDidItStart.title" />
           </H2>
+          {howdiditstart.edited && (
+            <H3 fontWeight="normal" paddingLeft="1rem">
+              <Trans id="confirmationPage.editedTag" />
+            </H3>
+          )}
           <EditButton
-            path="/howdiditstart"
             label="confirmationPage.howDidItStart.title.edit"
+            path="/howdiditstart"
+            edited={howdiditstart.edited}
           />
         </Flex>
 
-        {containsData(howdiditstart) ? (
+        {howdiditstart.howDidTheyReachYou.length ? (
```

Comment:
Is there a reason not to use containsData? It's being used everywhere

Reply:
containsData simply checks if the object contains anything. In this case if the field is flagged as edited it will render an empty confirmation summary. This way it will render "No information provided." unless a selection has been made.